Userland/Libraries/LibCore/System.cpp

```diff
@@ -1191,7 +1197,7 @@ ErrorOr<void> exec(StringView filename, ReadonlySpan<StringView> arguments, Sear
         envp[environment->size()] = nullptr;
 
         if (search_in_path == SearchInPath::Yes && !filename.contains('/')) {
-#    if defined(AK_OS_MACOS) || defined(AK_OS_FREEBSD)
+#    if defined(AK_OS_MACOS) || defined(AK_OS_FREEBSD) || defined(AK_OS_SOLARIS)
```

Comment:
generic BSD

Reply:
It's only MacOS,FreeBSD and Solaris that don't support execvpe and need that workaround.
OpenBSD and NetBSD support that just fine,no need to make it more complicated for them.